ui/src/components/PopInput/PopInput.jsx

```diff
@@ -117,80 +79,84 @@ export default class PopInput extends React.Component {
   }
 
   render() {
-    const linkClass = 'editable-click';
-    const busyVisibility = this.isBusy() ? '' : 'visibility-hidden';
-    const inputVisibility = !this.isBusy() ? '' : 'visibility-hidden';
```

Comment:
Removing this `visibility-hidden` class and using conditional rendering instead.